lib/samson/tasks/lock_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+module Samson::Tasks
+  class LockCleaner
+
+    def self.start
+      new.start
+    end
+
+    def start
+      task.tap(&:execute)
+    end
+
+    def task
+      @task ||= Concurrent::TimerTask.new(run_now: true, execution_interval: 60, timeout_interval: 10) do
+        Lock.remove_expired_locks
+      end.with_observer(self)
```

Comment:
Why do you say that? The `TimerTask` doesn't start running until  `execute` is called which is after the observer is added. `run_now: true` schedules the task to run immediately after `execute` instead of waiting until the first `execution_interval` occurs.